### ISSUE TITLE
App doesn't start - Couldn't register with accessibility bus: Did not receive a reply.

### ISSUE THREAD

**Issue by AlejandroC90** [rbrs/ubuntu-twitch-indicator#6]
It's me again, well, I had to reinstall archlinux and now when I try to run twitch-indicator, it shows me this, I searched for the problem but any solution worked for me, so I was wondering if it was some kind of dependence. 
![image](https://cloud.githubusercontent.com/assets/5132907/8188329/d47a717a-141a-11e5-8261-08c1c141bf15.png)

**Comment by rolandasb** [comment]
The error shows that it can't import AppIndicator3 which is in girl1.2-appindicator3-0.1 package.

**Comment by AlejandroC90** [comment]
Well, sorry for the late response, but I couldn't find the solution at the moment and was Very busy.
But today, I had to reinstall f.lux (I don't know if you know this program) and it installed some packages as dependences: libappindicator-gtk2 libappindicator-gtk3 libappindicator-sharp.
After that, I remembered what was the problem with twitch-indicator, so I decided to give it a try, and it works normal now, pretty interesting.
